typescript/infra/helm/warp-routes/templates/env-var-external-secret.yaml

```diff
@@ -22,6 +22,7 @@ spec:
       data:
         GCP_SECRET_OVERRIDES_ENABLED: "true"
         GCP_SECRET_OVERRIDE_{{ $.Values.hyperlane.runEnv | upper }}_COINGECKO_API_KEY: {{ printf "'{{ .%s_coingecko_api_key | toString }}'" .Values.hyperlane.runEnv }}
+        GCP_SECRET_OVERRIDE_HYPERLANE_{{ .Values.hyperlane.runEnv | upper }}_KEY_DEPLOYER: {{ print "'{{ .deployer_key | toString }}'" }}
```

Comment:
required because behind the scenes some of the getters in the monitor try to fetch this, despite it not being used. Not ideal, but not concerned imo